reproducibility_intro_activity.Rmd

```diff
@@ -0,0 +1,154 @@
+
+Note that you or a collaborator can now readily incorporate your suggestion by clicking `Commit suggestion` or they could comment back to further discuss.
+
+<input type="checkbox"> Click `Commit suggestion` and then `Commit changes`.  
+<input type="checkbox"> Return to the `Conversation` tab and scroll to the bottom. You should see an additional commit now!
+
+### 2b: Reading GitHub Actions logs
+
+On the bottom of your PR you will notice a box at the bottom that will probably either say `Some checks haven't completed yet` or `All checks passed`.  
+
+This is an automatic check run by GitHub Actions which is one of many systems used for Continuous Integration/Continuous Deployment.
+
+<input type="checkbox"> If needed, click `Show all checks`.  
+<input type="checkbox"> Click `Details` at the end of the line that says `Render R Example / render-notebooks (pull_request)`.  
+<input type="checkbox"> Click around in here to investigate what GitHub Actions logs look like.
+
+For more learning about using [GitHub Actions for science see our course](https://hutchdatascience.org/GitHub_Automation_for_Scientists/).
+
+### 2c: Updating your local repository
+
+If you followed the steps in the `Leaving comments and suggestions` section of this activity you will now have at least one commit that is on your online version of your repository but the changes done online won't automatically get to your local repository. To do this we need to `Pull` the latest changes to our computer.
+
+<input type="checkbox"> Return to GitHub Desktop.  
+<input type="checkbox"> Click the `Fetch origin` button.  
+<input type="checkbox"> You should be prompted to `Pull origin` click this button.  
+<input type="checkbox"> Now your changes should be synced. Open up the file you've changed with any default text editor to look for the change you made to make sure.  
+
+### 2d: Merging your PR
+
+In a real life setting the life of a Pull request could be long or short. What's important is that the discussions that need to be had about the changes are had before the pull request is merged.
+
+Pull requests and GitHub have their own kind of etiquette. Make sure you talk to your team or collaborators to create and discuss what kind of guidelines and workflows you'd like to adhere to so everyone can do their best work!
+
+<input type="checkbox"> Return to GitHub Pull Request Online.  
+<input type="checkbox"> Click the `Merge` button. Follow up with the `Confirm merge` button.  
+
+Yay! Congrats.
+
+### For more learning:
+- [Docs and Usability course](https://jhudatascience.org/Documentation_and_Usability/)
+- [Intro to Reproducibility course](https://jhudatascience.org/Reproducibility_in_Cancer_Informatics/introduction.html)
+- [Advanced Reproducibility course](https://jhudatascience.org/Adv_Reproducibility_in_Cancer_Informatics/introduction.html)
+- [Github Automation for Scientists course](https://hutchdatascience.org/GitHub_Automation_for_Scientists/)
```

Comment:
```suggestion
- [Github Automation for Scientists course](https://hutchdatascience.org/GitHub_Automation_for_Scientists/)
- [ITN Course "Containers for Scientists"](https://hutchdatascience.org/Containers_for_Scientists/)
```

Reply:
adding this one here too

Reply:
Edited the suggestion so the format is the same. Thanks to both of you!